event/trigger.go

```diff
@@ -0,0 +1,8 @@
+package event
+
+const (
+	// ETTriggerWorkflow signals that a workflow has been triggered
+	// Payload will be the `workflow.ID` of the workflow that should be run
+	// This event should block until the workflow has completed its run
+	ETTriggerWorkflow = Type("trigger:Workflow")
```

Comment:
categorical nesting error here. this should be `workflow:trigger` so we can add more `workflow`-namespaced events